cmds/xinit/xinit.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+var (
+	tczPackages  = []string{"Xorg-7.7-bin", "Xorg-7.7-dev", "Xorg-7.7-lib-dev", "Xorg-7.7-lib", "Xorg-7.7", "Xorg-fonts", "xorg-proto", "xorg-server-dev",
+"xorg-server","Xprogs","Xlibs","xpad-locale","xpad","Xlibs","Xfbdev","xvid","xkeyboard-config","xz","gdb","libX11-dev","libX11","libXfixes","i2c-4.8.17-tinycore64","graphics-4.8.17-tinycore64","libXvmc","libdrm","pixman","libXinerama","libXrandr","libXdamage","libXcursor","libXtst","libxshmfence","xf86-video-intel","openssh","aterm"}
+	sshdCommands = []string{"Protocol 2", "AcceptEnv LANG LC_*", "UsePAM no", "ChallengeResponseAuthentication no", "passwordauthentication no", "AuthorizedKeysFile ~/.ssh/authorized_keys", "PermitRootLogin without-password", "X11Forwarding yes", "RSAAuthentication yes", "PubkeyAuthentication yes", "X11DisplayOffset 10", "X11UseLocalhost yes"}
+)
+
+func setup() error {
+	if false {
+		return syscall.Mount("/tmp", "/tmp", "tmpfs", syscall.MS_MGC_VAL, "")
+	}
+	cmd := exec.Command("dhclient", "&")
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	err := cmd.Run()
+	if err != nil {
+		log.Printf(" error: %v. Continuing...", err)
+	}
+	fmt.Printf("Ip link below: \n")
+	cmd = exec.Command("ip", "link", "show")
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	err = cmd.Run()
+	if err != nil {
+		log.Printf(" error: %v. Continuing...", err)
+	}
+	return nil
+}
+
+func sshSetup() error {
+	sshdLoc := "/etc/ssh/sshd_config"
+	cmd := exec.Command("tcz", "openssh")
```

Comment:
same comment as dhclient -- we have tcz as go code, we should be able to call a function like

`tcz.GetPackage("openssh")`

where the function returns an error or something.

Reply:
that's a nice idea, and a tcz package is also a nice idea :-)